oasislmf/cli/config.py

```diff
@@ -29,6 +29,7 @@ class ConfigCmd(OasisBaseCommand):
     :source_accounts_profile_path: Source OED accouns (JSON) profile describing the financial terms contained in the source accounts file (optional)
     :source_exposure_file_path: Source OED exposure (CSV) file path
     :source_exposure_profile_path: Source OED exposure (JSON) profile describing the financial terms contained in the source exposure file (optional)
+    :summarise_exposure: Whether to generate exposure summary report
```

Comment:
It might be better to get rid of "whether etc.", and to add the file type to the help text, e.g.

   Generates an exposure summary report in JSON 

I think this reads better. I think there are probably other commands where we've got "whether" in the help text for boolean arguments, which we should also rewrite to make them simpler, but for the moment, we can just update this one.